recipe/meta.yaml

```
@@ -1,27 +1,36 @@
 {% set version = "8.0.0" %}
-{% set sha256 = "95305a886f2c42e9603c034ea684e5c031d9d4222c66ed6d85eb3ae15d631e4b" %}
 
 package:
   name: ipyparallel
   version: {{ version }}
 
 source:
-  fn: ipyparallel-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 95305a886f2c42e9603c034ea684e5c031d9d4222c66ed6d85eb3ae15d631e4b
 
 build:
-  number: 0
+  number: 1
   noarch: python
+  ignore_run_exports:
+    - nodejs
```

After trying a bunch of stuff locally, seems we needed this also

```suggestion
    - nodejs *
```